cli/src/diff_util.rs

```diff
@@ -749,6 +760,107 @@ pub fn show_diff_summary(
     })
 }
 
+struct DiffStat {
+    path: String,
+    added: usize,
+    removed: usize,
+}
+
+fn get_diff_stat(repo_path: String, left_content: &[u8], right_content: &[u8]) -> DiffStat {
```

Comment:
nit: rename `repo_path` to `ui_path` or just `path` since it's no longer a `RepoPath`

Reply:
Ah, good catch!